static/chats.js

```diff
@@ -162,6 +168,38 @@ function uzstadiVaardu(jaunaisVards) {
   return `${vecaisVards} kļuva par ${vards}`
 }
 
+function gStavoklis(teksts) {
+  let vardi = teksts.split(" ");
+  /* let izvads = vardi.slice(1); */
+ let izvads = Object.assign({}, vardi.slice(1));
+  
+  fetch('/chats/set_mood', {
+    method: 'POST',
+    headers: {
+        'Content-Type': 'application/json'
+    },
+    body: JSON.stringify({ "mood": izvads })
+});    
+
+gStavoklisNoFaila();
+
+}
+
+function gStavoklisNoFaila(){ 
+fetch('/chats/get_mood')
+.then(function(response) {
+  return response.json()
+})
+.then(function(newMood) {
+  console.log(newMood);
+ let saturs = Object.values(newMood);
+ console.log(saturs);
+ let satursV = Object.keys(newMood);
+ console.log(satursV);
+ setCookie('mood', saturs, 90)
```

Comment:
Jā, šis mums noder, ja mēs pārlādējam lapu, tad mums būs garastāvoklis.

Bet kā mēs varam zināt savu garastāvokli?

Savu zināt mēs ar doto risinājumu vienmēr varam (pieņemsim, ka augstāk aprakstītais nenotiks nekad reāli mūsu projektā). Mēs zinām savu, jo mēs to saglabājam uz 90 dienām pēc pierakstīšanas.

Piekļūt mēs tam varētu šādi:
augšā failā zem var vards definējam:
```js
let mood = getCookie('mood') || "Default mood"
```
Un šeit vienu rindiņu zemāk
mood = saturs, lai gan jāpadomā par labākiem mainīgo nosaukumiem.

Reply:
Šeit testēju, kā ar fetch metodi var ielasīt mainīgajā no serverī esošā faila datus, lai tos varētu izvadīt čata logā. Vispirms gribēju pārbaudīt kā var dabūt no serverī esošā faila objekta atslēgas un pēc tam objekta vērtības, tad gribēju šīs vērtības saglabāt mainīgajā un to mainīgo pielikt čatotāja vārdam. Sāku no vieglākā uz sarežģītāko, lai rastos izpratne kā strādā metode "fetch" , taču man vēl joprojām nav īsti saprašanas, kāpēc uzrakstot return saturs, vai return satursV, ārpus fetch metodes šos abus mainīgos nerāda, tāpat ka šo mainīgo saturu, vienīgais, kas man ienāca prātā, kā šo servera faila saturu dabūt - redzamu citām funkcijām - ierakstīt šo saturu sīkdatnē - cookie.